data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -828,10 +835,7 @@
             ai_type=return_guardian
             action=add
             [filter]
-                [filter_wml]
-                    upkeep=loyal
-                [/filter_wml]
-                type=Dwarvish Fighter
+                role=dwarvish_return_guard
             [/filter]
```

Comment:
I would prefer to use an `id` over `role`, because then you don't need an `[filter]` either, MAIs accept id's directly.